### PR TITLE
feat: update script

### PR DESCRIPTION
**Pull request by nnixaa**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -176,10 +173,19 @@ curl -k -L -o setup.sh https://raw.githubusercontent.com/uibakery/self-hosted/ma
...
+#cp ../setup.sh ./setup.sh
+#cp ../update.sh ./update.sh
+
 echo "Configuring application settings....."
-sudo bash ./setup.sh
+
+export SESSION_ID
+export GET_KEY_LINK
+export LICENCE_SERVER
+./setup.sh
```
Comment (nnixaa) [inline install.sh:185]: вот так можно передавать и не плодить переменные
зачем с судо вызывать сетап ваще не понятно, но тупо в файлы пишет

Comment (nnixaa) [inline install.sh:185]: This way we can pass variables from the main script.

File: setup.sh [hunk 2]
```diff
@@ -3,15 +3,17 @@ GREEN='\033[0;32m'
 CYAN='\033[0;36m'
 NC='\033[0m' # No Color
 
-
-printf "Enter license key(https://uibakery.io/get-an-on-prem-version-of-ui-bakery?utm_source=install_sh):\n"
```
Comment (nnixaa) [inline setup.sh:7]: This way link is can be clicked

File: setup.sh [hunk 3]
```diff
@@ -3,15 +3,17 @@ GREEN='\033[0;32m'
...
+printf '\n'
+printf "Enter license key:\n"
 while read license; do
   test "$license" != "" && break
   printf "${RED}License key is required!${NC}\n"
   printf "Enter license key:"
 done
 printf "License key: ${license}\n\n"
 
+curl -s -XPOST -H "Content-type: application/json" -d '{"event": "license", "session": "'"${SESSION_ID}"'", "key": "'"${license}"'"}' $LICENCE_SERVER  &> /dev/null
```
Comment (nnixaa) [inline setup.sh:16]: Now the curl output is ignored as well as "progress"

File: install.sh (hunk 1 above)
Comment (Tibing) [reply install.sh:185]: add this as a comment please

File: setup.sh (hunk 2 above)
Comment (Tibing) [reply setup.sh:7]: I was able to click it even in my way :D

Comment (Tibing) [reply setup.sh:7]: image
I can't click on this link. Let's just print the link in terminal

**Comment by Tibing** [comment]
image
Doesn't work. You have to call ./setup.sh with sudo

**Comment by Tibing** [comment]
let's add this to .gitignore ui-bakery-on-premise/

**Comment by Tibing** [comment]
image
Can't click, let's just print link in terminal

**Review by Tibing — APPROVED** [review]

**Pull request merged**